CMakeLists.txt

```diff
@@ -37,4 +37,13 @@ add_executable(tmatrix ${tmatrix_SRC})
 # Specify the include directory.
 include_directories(include)
 
+# Specify how to compress the man page.
+configure_file(${PROJECT_SOURCE_DIR}/tmatrix.6 ${PROJECT_BINARY_DIR}/tmatrix.6)
+add_custom_command(
+    OUTPUT tmatrix.6.gz
+    COMMAND gzip -f tmatrix.6
+    DEPENDS tmatrix.6)
+add_custom_target(manpage ALL DEPENDS tmatrix.6.gz)
+
```

Comment:
Can you add a comment here just saying `Install tmatrix binary and manpage.`? I forgot to add it when I added the install() command.

Reply:
Done.